app/src/main/java/pl/futurecollars/invoicing/controller/tax/TaxCalculatorApi.java

```diff
@@ -0,0 +1,18 @@
+package pl.futurecollars.invoicing.controller.tax;
```

Comment:
Jeśli masz pakiet o nazwie api to rozumiem, że powinno się to tam znaleźć dla zachowania konwencji